### DISCUSSION TITLE
希望可以保留UP主过滤功能

### DISCUSSION THREAD

**Discussion by nothingshiru** [magicdawn/Bilibili-Gate#113] [General]
看到说过滤跟拉黑功能重复，后期可能会删除。我觉得过滤和拉黑还是不一样的。拉黑会导致对方看不了我的主页，也无法跟我交流。我只是不喜欢有些人的视频出现在推荐，过滤功能很合适，拉黑就太过了
![屏幕截图 2024-07-20 102102](https://github.com/user-attachments/assets/8392c377-01f8-44c6-aa7f-fbd5e758339b)

**Comment by magicdawn** [comment]
相当于一个 warn, 当时写这段话是这么觉得的.
一般功能有人用, 而且不是特别难维护就会保留的.
